airbyte-integrations/connectors/source-tiktok-marketing/source_tiktok_marketing/streams.py

```diff
@@ -108,6 +114,14 @@ class TiktokStream(HttpStream, ABC):
     # max value of page
     page_size = 1000
 
+    def __init__(self, **kwargs):
+        super().__init__(authenticator=kwargs.get("authenticator"))
```

Comment:
Do we need to call `authenticator` directly here? I believe it should be passed automatically if we have such a property inside the passing args. Please check.